src/handle_wrap.cc

```diff
@@ -69,6 +81,7 @@ HandleWrap::HandleWrap(Environment* env,
   HandleScope scope(env->isolate());
   Wrap(object, this);
   env->handle_wrap_queue()->PushBack(this);
+  env->SetMethod(object, "isRefed", HandleWrap::IsRefed);
```

Comment:
It probably should be `isUnrefed()`, but `isRefed()` seems a lot simpler to think about.